tests/global_config.py

```diff
@@ -187,10 +187,12 @@ def _get_default_storage_class(sc_list):
 # If the DataImportCron uses a different prefix than the DataSource name
 # use data_import_cron_prefix in matrix dict to specify new prefix.
 auto_update_data_source_matrix = [
```

Comment:
templates are still supported. this drops coverage for templates. No?

Reply:
can we extend this for cluster_prefeence

Reply:
Yes, this is a good idea, but lets try and merge this first before we add an assertion on the used preferences, I don't want to make this PR too big and over extend it's purpose